pkg/agent/multicast/source_utils.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// The functions in the file are copied from golang source, maybe find a way to remove it
+//
+package multicast
```

Comment:
maybe move it to path third_party?

Reply:
Do you mean move the patched version of golang/sys to third_party https://github.com/ceclinux/sys/commit/eb11d17c505531acffcfbc05c9eb4d857d9bd156?

Reply:
moved